src/originalBudgetTrackingApp/configManager/configManager.ts

```diff
@@ -16,10 +16,10 @@ export interface Config {
     googleSheets?: GoogleSheetsConfig;
     ynab?: YnabConfig;
   };
-  scraping: {
-    'numDaysBack': number;
-    'showBrowser': boolean;
-    'accountsToScrape': AccountToScrapeConfig[];
+  accountsToScrape: AccountToScrapeConfig[];
+  globalConfig: {
+    numDaysBack: number;
+    showBrowser: boolean;
```

Comment:
@brafdlog Split the `accountsToScrape` from the other global config.

Reply:
The is a trade-off here. I want all the configs that configured by the `configEditor` in one sub-object, but it means that any different sub-object will need to take its config from the `globalConfig`.

And the opposite- you suggesting each config will be in its used object, and the `configEditor` will need to collect them to one view.

OK, we just need to take one of the options.

Reply:
I mean, I have no objection to your proposal, I'm working on it.